### ISSUE TITLE
when using title i get the name but also with some garbage

### ISSUE THREAD

**Issue by MadalaSM** [pytube/pytube#498]
when trying to get the title in hebrew of this: https://www.youtube.com/watch?v=g4t2GDIoquE
I get some garbage with it.

![Capture](https://user-images.githubusercontent.com/56499360/68533830-39bc9900-0336-11ea-8a87-d55a7771ae8b.PNG)

**Comment by flexiblestoic** [comment]
Yes, same issue. Encoding seems off. 

```python
from pytube import Playlist, YouTube

def readvideo():
    videourl = "https://www.youtube.com/watch?v=8EbLlgKFk9c&list=PL_6f0SGLJOb6siI6ALwCa0w_VJ9AV5PcC&index=2&t=0s"
    vid_title = YouTube(videourl).title
    print(vid_title)

readvideo()
```

Gets me : `I Still Haven&#39;t Found What I&#39;m Looking For - U2 (Gospel Soul Cover) ft. Rogelio Douglas, Jr.`

**Comment by flexiblestoic** [comment]
I quickly looked into the issue. Line 120 __main___.py we have

```python
            # Fix for KeyError: 'title' issue #434
            if 'title' not in self.player_config_args:
                i_start = (
                    self.watch_html
                    .lower()
                    .index('<title>') + len('<title>')
                )
                i_end = self.watch_html.lower().index('</title>')
                title = self.watch_html[i_start:i_end].strip()
                index = title.lower().rfind(' - youtube')
                title = title[:index] if index > 0 else title
                self.player_config_args['title'] = title
```
The title is not parsed before extraction, hence the garbled html coding for certain characters

A solution could be:

```python

            import html

            # Fix for KeyError: 'title' issue #434
            if 'title' not in self.player_config_args:
                i_start = (
                    self.watch_html
                    .lower()
                    .index('<title>') + len('<title>')
                )
                i_end = self.watch_html.lower().index('</title>')
                title = self.watch_html[i_start:i_end].strip()
                index = title.lower().rfind(' - youtube')
                title = title[:index] if index > 0 else title
                self.player_config_args['title'] = html.unescape(title)
```
